mobile/package.json

```diff
@@ -15,18 +15,18 @@
     "@buf/gnolang_dsocial-indexer.connectrpc_es": "^1.4.0-20240621073300-3f8442294fd1.3",
     "@buf/gnolang_dsocial-notification.bufbuild_es": "^1.10.0-20240621073240-68037cb72eee.1",
     "@buf/gnolang_dsocial-notification.connectrpc_es": "^1.4.0-20240621073240-68037cb72eee.3",
-    "@buf/gnolang_gnonative.bufbuild_es": "^1.9.0-20240430140846-fe9f3938584d.1",
```

Comment:
I'm just removing '@buf/gnolang_gnonative.bufbuild_es' and updating "@gnolang/gnonative".

The "expo-dev-client" was reorganized by npm.